utils/sampler/weighted_without_replacement.go

```diff
@@ -26,13 +26,3 @@ func NewWeightedWithoutReplacement() WeightedWithoutReplacement {
 		w: NewWeighted(),
 	}
 }
-
-// NewBestWeightedWithoutReplacement returns a new sampler
-func NewBestWeightedWithoutReplacement(
```

Comment:
I don't think we ever use this outside of tests. Should we delete the entire struct too since we're doing the same for `UniformBest` (or keep both)?

Reply:
Broke out the utils/sampler change here: https://github.com/ava-labs/avalanchego/pull/3219